CONTRIBUTING.md

```diff
@@ -38,10 +38,12 @@ It will help tremendously in avoiding comments and speeding up the PR process.
 
 Please view our contrib [README.md](contrib/README.md) for information on new integrations.
 
-### Go Modules
+### Dependencies
 
 When adding a new dependency, especially for `contrib/` packages, prefer the minimum secure versions of any modules rather than the latest versions. This is to avoid forcing upgrades on downstream users for modules such as `google.golang.org/grpc` which often introduce breaking changes within minor versions.
 
+The core non-`contrib/` packages also have their individual dependencies documented in `depware.txt` files in their respective directories. The `github.com/tailscale/depaware` tool lists the dependencies for a specific pacakge, rather than the module as a whole. The tool also identifies OS-specific dependencies and packages which use `unsafe` or `cgo`. The `depware.txt` files must be up-to-date for each commit so that dependency updates can be easily reviewed. Run `./depware.sh` in the repository root to update `depaware.txt` for the core packages.
```

Comment:
TODO: which Go version should be used to run `depaware`? It's important that the version used to generate `depaware.txt` and the version to check it are the same.

Reply:
I vote for latest supported release, but I don't have a strong opinion on this other than that it's easiest to understand.